sns/sns.go

```diff
@@ -0,0 +1,47 @@
+package sns
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/sns"
+)
+
+type SNSConfigAtrributes struct {
+	DeliveryPolicy            string `json:"DeliveryPolicy,omitempty"`
```

Comment:
Are the json tags needed here?

Reply:
It is being used to transform a struct into a `map[string]string`

But if it is not necessary it is probably faster to directly access the struct values. It's a bit hacky...